appveyor.yml

```diff
@@ -0,0 +1,90 @@
+# clone directory
+clone_folder: C:\MuseScore
+# fetch repository as zip archive
+# shallow_clone: true                 # default is "false"
+# set clone depth
+clone_depth: 3                      # clone entire repository history if not defined
+# environment variables
+# this is how to allow failing jobs in the matrix
+#matrix:
+#  fast_finish: true
+#  allow_failures:
+#    - platform: x86
+#      configuration: Release
+# build cache to preserve files/folders between builds
+cache:
+  - dependencies.7z
+  - C:\Qt\Tools\mingw530_32\lib
+  - C:\Qt\Tools\mingw530_32\i686-w64-mingw32
+  - C:\Program Files (x86)\Jack
+  - C:\Qt\Tools\mingw530_32\libexec\gcc\i686-w64-mingw32\5.3.0\cc1plus.exe
+  - C:\MuseScore\build.release\thirdparty
+  - C:\ccache
+#cache:
+#  - packages -> **\packages.config  # preserve "packages" directory in the root of build folder but will reset it if packages.config is modified
+#  - projectA\libs
+#  - node_modules                    # local npm modules
+#  - %APPDATA%\npm-cache             # npm cache
+# build platform, i.e. x86, x64, Any CPU. This setting is optional.
+platform:
+  - x86
+# build Configuration, i.e. Debug, Release, etc.
+configuration: Release
+# scripts to run before build
+before_build:
+  - IF NOT EXIST dependencies.7z (
+    start " " /wait "C:\MinGW\msys\1.0\bin\wget" --no-check-certificate "https://drive.google.com/uc?export=download&id=0BxjayMZiuupOSXF6a0ZDUG5rNDg" -O dependencies.7z &&
```

Comment:
Let's move this file to a more durable place... 
Use this URL http://utils.musescore.org.s3.amazonaws.com/musescore_dependencies_win32.7z